include/utils/Devirt.h

```diff
@@ -38,7 +41,9 @@ namespace llvm {
   class Devirtualize : public ModulePass, public InstVisitor<Devirtualize> {
```

Comment:
Question. Why are we using this old devirtualizator from DSA? I thought that seadsa has its own, no?

Reply:
As we discussed, sea-dsa doesn't have one.